packages/common/src/initiatives/HubInitiatives.ts

```diff
@@ -269,6 +272,7 @@ export async function enrichInitiativeSearchResult(
       thumbnail: "not-implemented",
       workspaceRelative: "not-implemented",
     },
+    location: deriveLocationFromItem(item),
```

Comment:
Ditto here, collaboration needs to add this function in the `itemToSearchResult` switch statement.